src/static/bitbucket/atlassian-connect.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Atlaskid",
+  "description": "Addon to display a 'release queue' panel in PRs",
+  "baseUrl": "https://lbatch.ngrok.io/",
```

Comment:
I'm just using my custom subdomain currently, will update once its running in micros.